src/css/wazimap-ng-v1.webflow.css

```diff
@@ -8282,3 +8282,9 @@ label {
   }
 }
 
+@media print {
+    .page-break-before {
+        page-break-before: always;
```

Comment:
this should be `break-before: page` `page-break-before` is an old property.

Reply:
done